app.rb

```diff
@@ -0,0 +1,66 @@
+require_relative './student'
+require_relative './teacher'
+require_relative './rental'
+require_relative './book'
+
+class App
+  attr_reader :book_list, :people_list
+
+  def initialize
+    @book_list = []
+    @people_list = []
+    @rental_list = []
+  end
+
+  def add_book(title, author)
+    Book.new(title, author)
+  end
+
+  def add_student(classroom, age, name, parent_permission)
+    student = Student.new(classroom, age, name, parent_permission)
+    @people_list.push({ value: student, type: 'Student' })
+  end
+
+  def add_teacher(specialization, age, name)
+    teacher = Teacher.new(specialization, age, name)
+    @people_list.push({ value: teacher, type: 'Teacher' })
+  end
+
+  def add_rental(date, book_num, person_num)
```

Comment:
The user is not able to create a rental when clicking on option `5`. Please make sure that the user is able to create a rental.